File: packages/components/basic-controls/source/controls/Dropdown/ThreeDotsMenuItem.ts
import { HTMLFormComponent } from 'fh-forms-handler';
import {DropdownItem} from "./DropdownItem";

class ThreeDotsMenuItem extends DropdownItem {
   constructor(componentObj: any, parent: HTMLFormComponent) {
        super(componentObj, parent);
    }
}

export {ThreeDotsMenuItem};

File: fhCoreLite/basicControls/src/main/java/pl/fhframework/model/forms/ThreeDotsMenuItem.java
package pl.fhframework.model.forms;

import pl.fhframework.annotations.Control;

@Control(parents = {Dropdown.class, Repeater.class}, canBeDesigned = true)
public class ThreeDotsMenuItem extends DropdownItem implements Iconable {
    public ThreeDotsMenuItem(Form form) {
        super(form);
    }
}
